blocks/editable/index.js

```diff
@@ -183,12 +183,22 @@ export default class Editable extends Component {
 				blocks = pasteHandler( childNodes );
 			}
 
-			if ( blocks.length ) {
-				// We must wait for TinyMCE to clean up paste containers after this
-				// event.
-				window.setTimeout( () => this.splitContent( blocks ), 0 );
-				event.preventDefault();
+			if ( ! blocks.length ) {
+				return;
 			}
+
+			// We must wait for TinyMCE to clean up paste containers after this event.
```

Comment:
I don't think TinyMCE is doing this asynchronously, so `setTimeout` is a bit of a hack. Instead can we make sure this logic is only triggered after TinyMCE completes its cleaning?